src/System.Text.Json/System/Text/Json/JsonReaderException.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.Text.Json
+{
+    public class JsonReaderException : Exception
+    {
+        public JsonReaderException() : base() { }
```

Comment:
Why do we need this type? i.e. when would this exception be handled programatically?